apps/constants/base.py

```diff
@@ -354,7 +354,10 @@
 # Contributions.
 FOUNDATION_ORG = 1  # The charities.id of the Mozilla Foundation.
 
-VERSION_BETA = re.compile('(a|alpha|b|beta|pre|rc)\d*$')
+VERSION_BETA = re.compile(r"""(a|alpha|b|beta|pre|rc) # Either of these
```

Comment:
Why the change to a multi-line expression? It fits on 1 line and the comment could be on the line above.